WordPress/Classes/ViewRelated/Views/RichTextView/UITextView+RichTextView.swift

```diff
@@ -1,8 +1,35 @@
 import Foundation
 
+// MARK: - Content summary support
 
 extension UITextView {
-    @objc func frameForTextInRange(_ range: NSRange) -> CGRect {
+
+    /// Returns a count of valid text characters.
+    var characterCount: Int {
```

Comment:
This might honestly be overkill for this, but I'd love to see this right next to the implementation of `wordCount` in `NSString+Helpers`. It's a PITA to update since it's in WPShared, but I think it'd benefit from being able to see that the two implementations are so similar to each other.

Reply:
This is valid feedback - thank you! 

I agree with the notion of moving this to `WordPressShared`; would you object if I moved it to `String+Helpers.swift` and added some clarifying comment referring to `wordCount` in `NSString+Helpers.h/.m`?

Reply:
Sounds good!